lib/inferno/repositories/requirements.rb

```diff
@@ -25,7 +25,7 @@ def insert_from_file(path)
             url: row[:url],
             requirement: row[:requirement],
             conformance: row[:conformance],
-            actor: row[:actor],
+            actors: row[:actors].split(',').map(&:strip),
```

Comment:
actors in the requirements excel may have a blank actor, e.g., for DEPRECATED requirements, so a null check should be added here. Or maybe better would be to skip loading of DEPRECATED requirements as a first up front check and then any inconsistencies with normal requirements don't need to be addressed during loading or use.

Reply:
Why don't deprecated requirements have actors?

Reply:
Not sure exactly what we were thinking when we removed the actors from deprecated requirements, but there are cases where they aren't there. I could add them back, but I do think that simply not loading deprecated requirements into the requirements database will prevent other issues like this from popping up.